### PR TITLE
Moved to sdcardio

### PR DESCRIPTION
**Pull request by evaherrada**

### CODE REVIEW

File: CircuitPython_JEplayer_mp3/code.py [hunk 1]
```diff
@@ -317,8 +317,7 @@ def choose_folder(self, base='/sd'):
 def mount_sd():
     """Mount the SD card"""
     spi = busio.SPI(board.SCK, MOSI=board.MOSI, MISO=board.MISO)
-    sd_cs = digitalio.DigitalInOut(board.SD_CS)
-    sdcard = adafruit_sdcard.SDCard(spi, sd_cs, baudrate=6000000)
+    sdcard = sdcardio.SDCard(spi, board.SD_CS)
```
Comment (dhalbert) [inline CircuitPython_JEplayer_mp3/code.py:320]: The default `baudrate` is 8000000. I'm not sure why the original code set it to 6000000. Do you have the parts to test it?
```suggestion
    sdcard = sdcardio.SDCard(spi, board.SD_CS, baudrate=6000000)
```

tagging @jepler here

File: Data_Logging_with_Feather_and_CircuitPython/code.py [hunk 2]
```diff
@@ -18,8 +16,18 @@
 
 SD_CS = board.D10
 spi = board.SPI()
-cs = digitalio.DigitalInOut(SD_CS)
-sd_card = adafruit_sdcard.SDCard(spi, cs)
+
+try:
+    import sdcardio
+
+    sdcard = sdcardio.SDCard(spi, SD_CS)
```
Comment (dhalbert) [inline Data_Logging_with_Feather_and_CircuitPython/code.py:23]: missing underscore
```suggestion
    sd_card = sdcardio.SDCard(spi, SD_CS)
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
One question, one typo.

File: CircuitPython_JEplayer_mp3/code.py (hunk 1 above)
Comment (evaherrada) [reply CircuitPython_JEplayer_mp3/code.py:320]: Yeah I do have them. I'll test it now.

Comment (evaherrada) [reply CircuitPython_JEplayer_mp3/code.py:320]: @dhalbert Just tested and it works.

**Review by dhalbert — APPROVED** [review]
Thanks!

**Pull request merged**